legend-engine-xts-relationalStore/legend-engine-xt-relationalStore-generation/legend-engine-xt-relationalStore-pure/src/main/resources/core_relational/relational/graphFetch/relationalGraphFetch.pure

```diff
@@ -121,7 +121,7 @@ function <<access.private>> meta::relational::graphFetch::executionPlan::getTupl
    if($dbType == DatabaseType.DB2,
     | 100,
     | if($dbType == DatabaseType.Snowflake,
-      | 1000,
```

Comment:
Should we even change this now? Ideally, we should determine it by the SQL stmt size / list size